src/routes/routes.js

```diff
@@ -0,0 +1,88 @@
+import { Navigate, useRoutes } from 'react-router-dom';
+
+import { useSelector } from 'react-redux';
+import AuthLayout from '../layout/Auth/AuthLayout';
+import LandingPageLayout from '../layout/LandingPage/LandingPageLayout';
+import ForgotPassword from '../pages/Auth/ForgotPassword/ForgotPassword';
+import SignIn from '../pages/Auth/SignIn/SignIn';
+import SignUp from '../pages/Auth/SignUp/SignUp';
+import NotFound404 from '../pages/NotFound404';
+import MyReservations from '../pages/LandingPage/MyReservation/MyReservation';
+import ReserveCars from '../pages/LandingPage/ReserveCars/ReserveCars';
+import ItemDetail from '../pages/UserDashboard/ItemDetail/ItemDetail';
+import Home from '../pages/LandingPage/Home/Home';
+import {
+  ACCOUNT,
+  SIGNIN,
+  SIGNUP,
+  FORGOT_PASSWORD,
+  MY_RESERVATIONS,
+  HOME,
+  RESERVE_CARS,
+  USERDASHBOARDHOME,
+  USERS_DASHBOARD,
+  ADD_NEW_CAR,
+  DELETE_CAR,
+  ITEM_DETAIL,
+  CONTACT,
+  NOTFOUND,
+} from './routeConstants';
+import UsersDashboardLayout from '../layout/UsersDashboard/UsersDashboardLayout';
+import DashboardHome from '../pages/UserDashboard/DashboardHome';
+import AddNewCar from '../pages/UserDashboard/AddNewCar/AddNewCar';
+import Contact from '../pages/LandingPage/Contact/Contact';
+import DeleteList from '../components/DeleteCars/DeleteList';
+// import LoadingText from '../pages/LoadingText';
+
+export default function Router() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Router= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪